app/models/move.rb

```diff
@@ -119,6 +119,10 @@ def rejected?
     cancellation_reason == CANCELLATION_REASON_REJECTED
   end
 
+  def nomis_event_id
```

Comment:
What's the reasoning behind last, here? I'm wondering whether we could flag the meaning better with a rename of this method: `most_recent_nomis_event_id`?

Reply:
this is slightly confusing as we have a `nomis_event_id` column on the model as well, so not sure if that is the most recent?

Reply:
Agree that this is confusing - it's a long story but this PR explains things. Once merged I can rebase and remove this custom method. https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/747